Helpers/RLMRealm+Notifications.m

```diff
@@ -65,4 +65,26 @@ - (void)deleteObjectsWithNotification:(id<NSFastEnumeration>)array
     [self deleteObjects:array];
 }
 
+#pragma mark - private helpers
+
+- (BOOL)rbq_containsObject:(RLMObject *)object {
+    if (!object) {
+        return NO;
+    }
+    
+    Class objectClass = [object class];
+    
+    RLMProperty *primaryKeyProperty = object.objectSchema.primaryKeyProperty;
+    
+    if (primaryKeyProperty) {
+        id primaryKeyValue = [object objectForKeyedSubscript:[objectClass primaryKey]];
```

Comment:
Calling the class is unnecessary here since you can just get the key with `primaryKeyProperty.name`